### ISSUE TITLE
The OTP data is not being removed from cache store

### ISSUE THREAD

**Issue by ankurk91** [tzsk/otp#227]
Hi, @tzsk 

It looks like the generated OTP stored in cache store (redis in my case), does not expire within the configured expiry interval 

https://github.com/tzsk/otp/blob/d25941dfb91df3e00ec1a1e4268c7fb40625defc/src/Otp.php#L53-L54

```bash
//dump($ttl);

 DateInterval {#5193
     interval: + 19234d 07:06:04.0,
   }
```
Notice that the `$ttl` is 19234 days, and it wont be removed unless developer manually run `forget()` method.

I also checked the TTL via redis-cli, and it is not working as expected.
So basically our Redis database will stop working due to low space.

**Comment by ankurk91** [comment]
Fixed in 
https://github.com/tzsk/otp/commit/e04263268b27afd7c7639c3a6ca926d5967da9e5
